src/styles/variables.scss

```diff
@@ -31,6 +31,8 @@ $font-size-12: 12px;
 // Line heights
 $line-height: 1.5;
 $line-height-14: 14px;
+$line-height-24: 24px;
```

Comment:
**[blocker]**

Both of these are unnecessary. They are used in places, where the line-heigh is already correct (the 1.5 line-height we have globally).

If there is something in the table that is overwriting our global line-height ,then you can use the $line-height variable here, instead of $line-height-24 and line-height-21